src/Audio.md

```diff
@@ -0,0 +1,110 @@
+# Audio Overview
+
+Game Boy audio is sometimes called "8-bit", but this doesn't mean much: the term is used to designate audio similar to that produced by 8-bit consoles.
+
+A better description would be "chiptune": tunes produced by a chip.
+This section describes how audio is generated on the Game Boy.
+
+## Architecture
+
+{{#include imgs/apu_overview.svg:2:}}
+
+The Game Boy has four sound generation units, called **channels** 1 through 4, notated "CH1", "CH2", etc.
+Unlike some other sound chips, such as the SNES' S-DSP or the Atari 5200's POKEY, each sound channel is specialized in a way largely different from the other channels.
+
+Each channel generates an electronic signal; these signals are then mixed into two new channels (for stereo: one for the left ear, one for the right ear), which are then individually amplified, and then output either to the headphone jack, or the speaker[^speaker_mono].
+
+Channels 1 and 2, the "pulse channels", produce [pulse (rectangular) waves](https://en.wikipedia.org/wiki/Pulse_wave).
```

Comment:
Not sure about calling it rectangular wave. This seems like a more academic terminology, and is in my experience not commonly used in synthesis for music. (A legacy from Martin's "quadrangular" waves?) I would rather say something like "produce [pulse width modulated waves](https://en.wikipedia.org/wiki/Pulse-width_modulation) with 4 fixed pulse width settings."

Reply:
So this?
```suggestion
Channels 1 and 2, the "pulse channels", produce [pulse width modulated waves](https://en.wikipedia.org/wiki/Pulse-width_modulation) with 4 fixed pulse width settings.
```

Reply:
Yes.